crates/bdk/src/wallet/export.rs

```diff
@@ -268,13 +262,14 @@ mod test {
     #[test]
     #[should_panic(expected = "Incompatible change descriptor")]
     fn test_export_no_change() {
-        // This wallet explicitly doesn't have a change descriptor. It should be impossible to
+        // This wallet has a bad change descriptor (no glob). It should be impossible to
```

Comment:
"bad" is arbitrary. Can't we be more technical here.

Reply:
The technical reason in this case is "no glob". The absence of a glob `*` a.k.a. "wildcard" in the change descriptor is enough for `export_wallet` to hit the expected error.